script/Arcueid/Day 8/08_00_ARC08_8_1B.txt

```diff
@@ -126,17 +140,22 @@ Regardless of her explanation, I'm starting to get a handle on the situation.
 [sha:2910cc7549526b9a3e0bf95f405450b33c03256d]{
 -- Page 553, Offset 21165.
 -- 志貴クンは無神論者でしょ？　だから実感は湧かないだろうけど、私たち教会は主の御名のもと、人の善き営みを守る意思がある。たとえそれが、こんな末法の国でもね」
-You're an atheist, right? So it might be hard to believe, but our Church is dedicated to protecting the good of mankind in the name of the Lord. Even in a country under the latter day of the Dharma."
+// 末法 is Latter Day of the Dharma but uh that seems a bit obtuse as a reference to me?
+You're an atheist, right?
+ So this may not have sunk in yet,
+ but those of the Church are dedicated to protecting the works of man in the name of our Lord.
+ Even in a decadent country, like the one we're in now."
```

Comment:
I don't really approve of this change. [The Latter Day of the Dharma](https://en.wikipedia.org/wiki/Three_Ages_of_Buddhism#Latter_Day_of_the_Dharma) is a pretty important concept in Buddhism afaik. Moreover, I think "decadent" is a little soft. The English definitions I could find that didn't use the word Dharma used the phrase ["degenerate"](https://encyclopediaofbuddhism.org/wiki/Three_Ages_of_Dharma).

Reply:
Fair enough, we can keep it. Religious terms are fair game from a Church person I suppose